tests/generation/test_utils.py

```diff
@@ -2792,6 +2160,7 @@ def test_transition_scores_group_beam_search_encoder_decoder(self):
 
         self.assertTrue(torch.allclose(transition_scores_sum, outputs.sequences_scores, atol=1e-3))
 
+    @slow
```

Comment:
Any reason to tag this as `@slow`? :) (GPT2 is fairly cheap to run)

Reply:
oh, really. Okay, I just thought gpt-2 has to be slow. Will get it back 😃 
Also the PR still needs clean-up in the tests that override GenerationTesterMixin apparently